util/wren.mk

```diff
@@ -103,9 +103,13 @@ else
 	SHARED_LIB_FLAGS := -Wl,-soname,$@.so
 	SHARED_EXT := so
 
+ifeq (,$(findstring Windows,$(OS)))
```

Comment:
Okay, I don't have strong objection. :)
What you want? Change the part to below?
```
ifeq ($(OS),mingw32)
```